capabilities/weather/get-weather/profile.supr

```diff
@@ -0,0 +1,30 @@
+name = "weather/get-weather"
```

Comment:
As we suggest one use case per profile, it might be weird, but to get well formated profile name in catalog, profile description must be provided as well.

<img width="543" alt="image" src="https://user-images.githubusercontent.com/959390/133148945-e734f75a-ec07-4a4f-9343-14a94e0166af.png">


```suggestion
"""
Get Weather
Get weather information for selected location
"""

name = "weather/get-weather"
```

Reply:
One thing about the name, since this is specific to current weather and location by city, shouldn't the profile name reflect that as well? E.g. `weather/city-current` or something like that. A variations would be `weather/coords-current`, `weather/forecast-city` etc.

And similar for the usecase name (perhaps `GetCurrentWeatherInCity`).

Or is this too much future proofing?

Reply:
I like it 👍